projects/core/components/link/link.style.less

```diff
@@ -124,6 +124,8 @@
         padding-bottom: 0.15em;
         text-decoration: underline dashed;
         text-underline-offset: 0.2em;
+        // TODO: research if it can be used in rem units
+        /* stylelint-disable-next-line */
         text-decoration-thickness: 0.7px;
```

Comment:
it cannot be used in rem units (

Can stylelint - plugin exclude float values between [-5, 5] (not only integers) ?

Reply:
not supported (( only fixed values